sdk/inc/azure/platform/az_mqtt_mosquitto.h

```diff
@@ -48,6 +48,11 @@ typedef struct
    * @brief Handle to the underlying MQTT implementation (Mosquitto).
    */
   struct mosquitto* mosquitto_handle;
+
+  /**
+   * @brief Whether to use TLS for the underlying MQTT implementation.
+   */
+  bool use_tls;
```

Comment:
why is this optional?

Reply:
To make it easier to debug the MQTT connection